src/pages/login.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bella Coola</title>
+    <link rel="stylesheet" href="../../styles.css" />
+  </head>
+  <body>
+    <div class="app-outer-layout">
+      <div class="app-inner-layout login">
+        <div class="login-topside">
+          <img src="../assets/icons/app-large-icon.png" alt="icon" />
+          <div class="login-buttons">
+            <button id="login">Login</button>
+            <button id="signup">Sign-up</button>
+          </div>
+        </div>
+        <div class="login-signup-menu">
+          <form class="login-form">
+            <div class="email-container">
+              <label for="email"
+                >Email
+                <address></address
+              ></label>
+              <input type="email" name="email" id="email" />
+            </div>
+            <div class="password-container">
+              <label for="password">Password</label>
+              <input type="password" name="password" id="password" />
+            </div>
+            <a href="#">
+              <span class="forgot-password">Forgot passcode?</span></a
+            >
+          </form>
+          <a href="./login-menu.html">
```

Comment:
Y button in a?

Reply:
@docdrue there are two things. 
1. Semantics: the element we see on the page looks like  button and behaves like button, therefore I would like to consider it to be a button, rather than a link.
 
2. Reusability: in the future I might need to have the same component but with different functionality ( e.g. calling a script), if the element you mentioned was just a link, I'd have to transfer all the styles to a new button element. 

Overall, I believe that having a styled button component, wrapped in an anchor tag is a rather concise solution.